recipes/provenance/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "provenance" %}
+{% set version = "0.13.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7561c1fe46b1df559d632fa02b5dfb3539b2a76abf36a2b3e96fa03d0f43083a
+
+build: 
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - pytest 
+    - pytest-runner
+  run:
+    - alembic >=0.9.1
+    - boltons >=16.5.1
+    - cloudpickle >=0.2.1
+    - frozendict
+    - graphviz
+    - joblib ==0.14.1
+    - memoized-property >=1.0.2
+    - numpy
+    - ordered-set >=2.0.1
+    - paramiko
+    - psutil >=5.0.0
+    - psycopg2
+    - pyarrow
+    - python >=3.5
+    - python-graphviz
+    - s3fs >=0.0.9
```

Comment:
The wheel on pypi has:
```
Requires-Dist: s3fs (==0.0.9)
```
The constraint here is much loose. Is that okay?

Reply:
I tested it on later s3fs version, and things seem to work fine, but it's probably a good idea to match the wheel's requirements.